iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -31,23 +31,23 @@ internal sealed class MqttTransportHandler : TransportHandler, IDisposable
 
         private const string DeviceToCloudMessagesTopicFormat = "devices/{0}/messages/events/";
         private const string ModuleToCloudMessagesTopicFormat = "devices/{0}/modules/{1}/messages/events/";
-        private string _deviceToCloudMessagesTopic;
-        private string _moduleToCloudMessagesTopic;
+        private readonly string _deviceToCloudMessagesTopic;
```

Comment:
The only changes in this file are making some fields readonly and moving the const and readonly fields to the top of the class definition